README.md

```diff
@@ -61,17 +59,31 @@ Usage: chusaku [options]
 
 ```
 
+### Rake usage
+
+If you'd like to use Chusaku as a Rake task, add the following line to your `Rakefile`:
+
+```ruby
+Chusaku.load_tasks
+```
+
+This will then allow you to call:
+
+```sh
+bin/rake chusaku
+```
+
+To pass flags, pass them like you would from the CLI executable:
+
+```sh
+bin/rake chusaku -- --dry-run --exit-with-error-on-annotation
```

Comment:
From what I've seen working in the Ruby app space generally env variables are used for Rake tasks, but if you've confirmed this works I think it's probably fine to just ship it as-is for now since I don't imagine there'll be a reason to be passing lots of options around, and worse case it can be changed later if you get feedback that it's actually a problem (and that feedback will be very valuable for future)